x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,65 @@
+.appSearchResultContainer {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  width: 100%;
+}
+
+.appSearchResult {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
```

Comment:
Isn't `display: grid` just overriding `display: flex` immediately? It looks like we should delete lines 14-15, right?

Reply:
Yeeeessss. This was an oversight on my part.

Most of the styles were defined in the old code base as having a base flex layout, and then conditionally apply grid layouts if they were supported by the browser.

I of course did not want to continue maintaining two different layout patterns so I opted to use the grid layout, since all modern browsers appear to support it.

Here I believe I just forgot to remove the old flex layout.